pytket/docs/changelog.rst

```diff
@@ -1,13 +1,22 @@
 Changelog
 =========
 
-Unreleased
-----------
+1.36.0 (unreleased)
```

Comment:
Prefer to keep this as "Unreleased" only. We don't know if it will be 1.36.0 or 1.35.1 ...

Reply:
done in [37f0f1b](https://github.com/CQCL/tket/pull/1691/commits/37f0f1bef4fd7cda549be65d849e3b5af21282b4)